invites/async/reply.js

```diff
@@ -27,14 +27,13 @@ module.exports = function (server) {
 
     getInvite(reply.branch, (err, invite) => {
       if (err) return callback(err)
+      const { value: { content: { recps } } } = invite
       var whoami = server.whoami()
```

Comment:
:fire: this whoami is gonna cause trouble I think.
whoami is sync when run serverside
but when you access it over ssb-client (from the UI) it _has to be_ async .... this is a major gotcha which has kicked my arse heaps.

luckily can convert this to `server.id` here

Reply:
:white_check_mark: done